src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+    - `<organization_id>`:your organization's ID.
+    - `<api-key>`: The API key of a service account with the Org Admininstrator role.
+
+1. Optionally, you can now delete the provisioning service account. In the remainder of this page, replace `<api-key>` with the API key for the folder-management service account.
+
+## Create a folder and grant access to it
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin)
+
+1. To create a folder at the organization level, set its `parent_id` field to `root`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/folders' \
+           --header 'Authorization: Bearer <api-key>' \
+           --data '{"name" : "<folder_name>", "parent_id": "root"}'
+    ~~~
+
+    Replace:
+    - `<folder_name>`: a name for the folder.
+    - `<api-key>`: your service account's API key.
+
+    The response includes the new folder's ID.
+
+1. To create a descendent folder within a folder, set `parent_id` to the folder's ID:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/folders' \
+           --header 'Authorization: Bearer <api-key>' \
+           --data '{"name" : "<folder_name>", "parent_id": "<parent_id>"}'
+    ~~~
+
+    Replace:
+    - `<folder_name>`: a name for the folder.
+    - `<api-key>`: your service account's API key.
+    - `<parent-id>`: the ID of the folder that will contain the new folder.
+
+1. To grant your service account the `FOLDER_ADMIN` role on a folder and its descendent folders:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<service-account-id>/FOLDER/<folder-id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
+    - `<service-account-id>`: The ID of the service account you created in [Gather Required Information](#gather-required-information).
+    - `<folder-id>` with the ID of the folder.
+    - `<api-key>`: your service account's API key.
+
+    {{site.data.alerts.callout_info}}
+    You can also grant the role to a human user. However, during Limited Access, only a service account using the {{ site.data.products.db }} API or Terraform provider can manage folders.
+    {{site.data.alerts.end}}
+
+1. If you grant the role at the level of the organization, it is inherited on all folders. To give your service account the `FOLDER_MOVER` role at the organization level:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<service-account-id>/ORGANIZATION/<organization-id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
```

Comment:
I would suggest the following here:
Replace:
    - `<service-account-id>`: the service account ID (the role grantee)
    - `<organization-id>` with the ID you gathered in [Gather Required Information](#gather-required-information).
    - `<api-key>`: the role granter's API Key (not the API Key of the role grantee)

Reply:
DOne